.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
```

Comment:
Kādus failus tu šeit esi iepushojis?

Reply:
Aaa, tie man mapītee bija, jo es kaut kā tu caur citam programmam mēģināju mergot tos zarus. Tapec tur izveidojas. Tas traucē?

Reply:
.vs, bin un obj ir ignorējamie faili. Ja tev tie šeit parādās, tas nozīmē, ka kkas nav ok.